src/components/MessageList/MessageList.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Why?

Reply:
i added  "parser": "@babel/eslint-parser" and  its dependencies 
 then i was getting  a error Unexpected dangling '_' in '_id'.eslint[no-underscore-dangle](https://eslint.org/docs/rules/no-underscore-dangle)   
to fix that issue I added this comment this error usually occurs when you are using an underscore (_) at the beginning of a variable or property name, which is a naming convention that is generally discouraged in JavaScript.

screenshot:
![image](https://user-images.githubusercontent.com/99081689/210525747-656cb0d8-6510-45d5-ac07-c637da5c8bbe.png)

Reply:
i saw one comment in ChatBody>ChatBody.js file ` /* eslint-disable no-shadow */  `so i thought its ok to add those comments in file to fix this linting error

Reply:
![image](https://user-images.githubusercontent.com/99081689/210960141-e50d6ed4-b545-4d8e-bf6b-42a142aabf04.png)

it's showing a linting error on messages, anonymousmode  that 'messages' is already declared in the upper scope